### PR TITLE
Bump pgautofailover-enterprise version to 1.3.1

### PR DESCRIPTION
**Pull request by JelteF**
This also adds encryption of the packge as we have for pg_auto_failover.

### CODE REVIEW

**Comment by hanefi** [comment]
I wonder why the CI job did not get triggered. I want to see the outputs of the packaging scripts here

File: debian/package-postinst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eu
+
+pg_version=
+installation_message="
++--------------------------------------------------------------+
```
Comment (hanefi) [inline debian/package-postinst:7]: nit; I guess we can have 2 extra dashes here as the text width is wider than the seperator lines

**Comment by hanefi** [comment]
Some lines from CI jobs that caught my eye: 

EL/7 https://travis-ci.org/github/citusdata/packaging/jobs/684371231#L653
```
pg-auto-failover-enterprise13_10.x86_64: W: invalid-license Proprietary
```


Debian/Jessie Lintian output https://travis-ci.org/github/citusdata/packaging/jobs/684371235#L665-L669
```
W: postgresql-12-auto-failover-enterprise-1.3: binary-without-manpage usr/bin/pg-auto-failover-enterprise-pg-12-setup
W: postgresql-11-auto-failover-enterprise-1.3: binary-without-manpage usr/bin/pg-auto-failover-enterprise-pg-11-setup
W: postgresql-10-auto-failover-enterprise-1.3: binary-without-manpage usr/bin/pg-auto-failover-enterprise-pg-10-setup
E: pg-auto-failover-cli-enterprise-1.3: versioned-provides pg-auto-failover-cli-enterprise (= 1.3.1-1)
E: pg-auto-failover-cli-enterprise-1.3: dir-or-file-in-opt opt/pg_autoctl.gpg
```

debian/stretch https://travis-ci.org/github/citusdata/packaging/jobs/684371244#L371-L373
```
W: postgresql-11-auto-failover-enterprise-1.3-dbgsym: debug-file-with-no-debug-symbols usr/lib/debug/.build-id/5a/22e4612d57d00f6128305c99fe0e117868ddc3.debug
W: postgresql-12-auto-failover-enterprise-1.3-dbgsym: debug-file-with-no-debug-symbols usr/lib/debug/.build-id/fb/b1424305a853f5b118caf61e82ff186b274ae5.debug
W: postgresql-10-auto-failover-enterprise-1.3-dbgsym: debug-file-with-no-debug-symbols usr/lib/debug/.build-id/bd/d527b850e508a0cb0251009357a3b5d2046a04.debug
```

I should note that some of these messages appear in multiple distros.

Some funny ones:

el/6 https://travis-ci.org/github/citusdata/packaging/jobs/684371238#L627-L628
```
pg-auto-failover-enterprise13_11.x86_64: W: spelling-error Summary(en_US) Citus -> Cetus, Titus, Vitus
pg-auto-failover-enterprise13_11.x86_64: W: spelling-error %description -l en_US Postgres -> Postures, Postgraduates, Postgraduate
```

**Review by hanefi — APPROVED** [review]
Please go over the warnings and errors that I compiled in [my comment](https://github.com/citusdata/packaging/pull/473#issuecomment-625706130) and decide if there is some that needs attention

I also have 2 minor comments

File: debian/package-postinst (hunk 1 above)
Comment (JelteF) [reply debian/package-postinst:7]: In the actual output it's not wider, since `$pg_version` is replaced with 10/11/12.

**Pull request merged**